<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QPushButton" name="b1">
      <property name="text">
       <string>Push Me!!</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Sample Size</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Decimation</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="spinBox_dec">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1024</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_sampsize">
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>65000</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="6">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="PlotWidget" name="pyqtgraph_widget" native="true"/>
      </item>
      <item>
       <widget class="PlotWidget" name="pyqtgraph_widget_2" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_channel1">
        <property name="text">
         <string>Channel 1 Enable</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_channel2">
        <property name="text">
         <string>Channel 2 Enable</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Data Rate MB/s</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Queue Depth</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLCDNumber" name="lcdNumber_queuedepth">
        <property name="digitCount">
         <number>8</number>
        </property>
        <property name="mode">
         <enum>QLCDNumber::Dec</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLCDNumber" name="lcdNumber_datarate">
        <property name="digitCount">
         <number>8</number>
        </property>
        <property name="mode">
         <enum>QLCDNumber::Dec</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Lost</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLCDNumber" name="lcdNumber_lost">
        <property name="digitCount">
         <number>8</number>
        </property>
        <property name="mode">
         <enum>QLCDNumber::Dec</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
